homeassistant/components/webostv/media_player.py

```diff
@@ -231,17 +226,24 @@ def name(self):
     @property
     def state(self):
         """Return the state of the device."""
-        return self._state
+        client_state = self._client.power_state.get("state")
+        if client_state in [None, "Power Off", "Suspend", "Active Standby"]:
+            return STATE_OFF
+
+        return client_state
```

Comment:
We need to map the webostv client state to a valid home assistant state. Use a dictionary.

Reply:
Ah ok, the valid states are the ones here?
https://github.com/home-assistant/home-assistant/blob/ff7d2ac681ab085f74ca167d8e07f525d3287e60/homeassistant/const.py#L198-L225

Reply:
Out of those only on, off, playing, paused, idle and standby are relevant for the media_player domain.